internal/job/git_test.go

```diff
@@ -256,12 +259,12 @@ func (r *mockShellRunner) Run(_ context.Context, cmd string, args ...string) err
 
 func (r *mockShellRunner) RunWithOlfactor(
 	_ context.Context,
-	_ string,
+	_ []string,
 	cmd string,
 	args ...string,
-) error {
+) (*olfactor.Olfactor, error) {
 	r.got = append(r.got, append([]string{cmd}, args...))
-	return nil
+	return nil, nil
```

Comment:
Because this is just a mock, I think it's okay to return a `nil` Olfactor.

Reply:
Feels like, if this became a problem, it would be quick to find.